client/reader/discover/_style.scss

```diff
@@ -1,73 +1,89 @@
-.discover-attribution {
-	font-size: 13px;
-	font-family: $sans;
-	text-transform: lowercase;
-	color: $gray;
-	position: relative;
-	margin-top: 13px;
-	margin-bottom: 8px;
+.reader__card.card, .reader__full-post {
 
-	&.is-site {
-		margin-top: 15px;
-	}
+	.discover-attribution {
+		font-size: 13px;
+		font-family: $sans;
+		text-transform: lowercase;
+		color: $gray;
+		position: relative;
+		margin-top: 13px;
+		margin-bottom: 8px;
+
+		a {
```

Comment:
There's a preference to use specific classnames instead of generic selectors, although I think in this case there are already specific classes for links under `.discover-attribution`?